src/Aspire.Hosting.Yarp/NodeStaticBuildOptions.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Aspire.Hosting.Yarp;
+
+/// <summary>
+/// Configuration options for Node.js static build in YARP npm resources.
+/// </summary>
+[Experimental("ASPIREHOSTING001", UrlFormat = "https://aka.ms/dotnet/aspire/diagnostics#{0}")]
+internal sealed class NodeStaticBuildOptions
+{
+    /// <summary>
+    /// Gets or sets the package manager to use (e.g., "npm", "pnpm", "yarn").
+    /// </summary>
+    public string PackageManager { get; set; } = "npm";
+
+    /// <summary>
+    /// Gets or sets the install command (e.g., "install", "ci", "install --frozen-lockfile").
+    /// </summary>
+    public string InstallCommand { get; set; } = "install";
```

Comment:
```suggestion
    public List<string> InstallCommand { get; set; } = "install";
```

Might make constructing these easier? Especially if people want to conditionally configure the build command based on certain conditions.